docker-compose-dependencies.yml

```diff
@@ -16,7 +16,7 @@ services:
       - microserviceKeys_ccd_case_document_am_api=AAAAAAAAAAAAAAAA
       - microserviceKeys_am_role_assignment_service=AAAAAAAAAAAAAAAA
       - TESTING_SUPPORT_ENABLED=true
-      - JAVA_TOOL_OPTIONS=-Xms128m -Xmx512m -XX:MaxPermSize=512m
+#      - JAVA_TOOL_OPTIONS=-Xms128m -Xmx512m -XX:MaxPermSize=512m
```

Comment:
why did comment out?

Reply:
It was causing an error in the "service-auth-provider-api" docker container causing it to quit. Something along the lines of JAVA_TOOL_OPTIONS option not recognised.

Reply:
is it related to postgres update ?

Reply:
no, it was happening beforehand anyway. Maybe this change should not be a part of this PR though?

Reply:
normally yes, it is a fix.